### DISCUSSION TITLE
Unable to apply for Github student pack, I have a dated clg id but it has clg name is on other side and date is on other side

### DISCUSSION THREAD

**Discussion by parthshah2003** [community/community#53536] [GitHub Education]
Hello here,
I have a dated clg id but it has clg name is on other side and date is on other side

So i even uploaded it through camera but it only allows one side, even if i upload a merged screenshot of both sides it keeps autorejecting :(
Pls consider helping me

**Comment by samus-aran** [comment]
Hi @parthshah2003 👋 

We acknowledge that some institutions do not have dated Student ID (on the same side) so we would recommend that you acquire another piece of dated evidence and post in alongside your Student ID (including name, school name etc).

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list. If you have multiple pieces in the same image then select the higher option out of your submitted evidence types.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope this helps! Let us know how you get along 👍
